react-router/react-router-0.13.3.d.ts

```diff
@@ -0,0 +1,354 @@
+// Type definitions for React Router 0.13.3
+// Project: https://github.com/rackt/react-router
+// Definitions by: Yuichi Murata <https://github.com/mrk21>, Václav Ostrožlík <https://github.com/vasek17>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+///<reference path='../react/react.d.ts' />
+
+declare module ReactRouter {
+    import React = __React;
+
+    //
+    // Transition
+    // ----------------------------------------------------------------------
+    interface Transition {
+        path: string;
+        abortReason: any;
+        retry(): void;
+        abort(reason?: any): void;
+        redirect(to: string, params?: {}, query?: {}): void;
+        cancel(): void;
+        from: (transition: Transition, routes: Route[], components?: React.ReactElement<any>[], callback?: (error?: any) => void) => void;
+        to: (transition: Transition, routes: Route[], params?: {}, query?: {}, callback?: (error?: any) => void) => void;
+    }
+
+    interface TransitionStaticLifecycle {
+        willTransitionTo?(
+            transition: Transition,
+            params: {},
+            query: {},
+            callback: Function
+        ): void;
+
+        willTransitionFrom?(
+            transition: Transition,
+            component: React.ReactElement<any>,
+            callback: Function
+        ): void;
+    }
+
+    //
+    // Route Configuration
+    // ----------------------------------------------------------------------
+    // DefaultRoute
+    interface DefaultRouteProp {
+        name?: string;
+        handler: React.ComponentClass<any>;
+    }
+    interface DefaultRoute extends React.ReactElement<DefaultRouteProp> {}
+    interface DefaultRouteClass extends React.ComponentClass<DefaultRouteProp> {}
+
+    // NotFoundRoute
+    interface NotFoundRouteProp {
+        name?: string;
+        handler: React.ComponentClass<any>;
+    }
+    interface NotFoundRoute extends React.ReactElement<NotFoundRouteProp> {}
+    interface NotFoundRouteClass extends React.ComponentClass<NotFoundRouteProp> {}
+
+    // Redirect
+    interface RedirectProp {
+        path?: string;
+        from?: string;
+        to?: string;
+    }
+    interface Redirect extends React.ReactElement<RedirectProp> {}
+    interface RedirectClass extends React.ComponentClass<RedirectProp> {}
+
+    // Route
+    interface RouteProp {
+        name?: string;
+        path?: string;
+        handler?: React.ComponentClass<any>;
```

Comment:
And it is named so https://github.com/sergey-buturlakin/DefinitelyTyped/blob/master/react-router/react-router.d.ts